README.md

```diff
@@ -21,50 +21,66 @@ This is an app that allows a customer to view a takeaway menu and place an order
 How to run
```

Comment:
I find this really useful - definitely keep this level of detail up :)

I managed to figure out your implementation just from the README :) 

This is nice and clear for the user :)